src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,161 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    private Node<T> first;
+    private Node<T> last;
+    private int size;
+
+    public MyLinkedList() {
+        size = 0;
```

Comment:
FYI: since size is a primitive it will have default value of 0, so basically you don't need this constructor

Reply:
true, thanks